tensorboard/backend/event_processing/event_accumulator_test.py

```diff
@@ -582,6 +582,80 @@ def testTFSummaryTensor(self):
     self.assertTrue(np.array_equal(vector, [1.0, 2.0, 3.0]))
     self.assertTrue(np.array_equal(string, six.b('foobar')))
 
+  def _testTFSummaryTensor_SizeGuidance(self,
+                                        plugin_names,
+                                        tensor_size_guidance,
+                                        steps,
+                                        expected_count):
+    event_sink = _EventGenerator(self, zero_out_timestamps=True)
+    writer = tf.summary.FileWriter(self.get_temp_dir())
+    writer.event_writer = event_sink
+    with self.test_session() as sess:
+      summary_metadata = tf.SummaryMetadata()
+      for plugin_name in plugin_names:
+        summary_metadata.plugin_data.add(plugin_name=plugin_name, content='{}')
+      tf.summary.tensor_summary('scalar', tf.constant(1.0),
+                                summary_metadata=summary_metadata)
+      merged = tf.summary.merge_all()
+      for step in xrange(steps):
+        writer.add_summary(sess.run(merged), global_step=step)
+
+
+    accumulator = ea.EventAccumulator(
+        event_sink, tensor_size_guidance=tensor_size_guidance)
+    accumulator.Reload()
+
+    tensors = accumulator.Tensors('scalar')
+    self.assertEqual(len(tensors), expected_count)
+
+  def testTFSummaryTensor_SizeGuidance_DefaultToTensorGuidance(self):
+    self._testTFSummaryTensor_SizeGuidance(
+        plugin_names=['jabberwocky'],
```

Comment:
> It takes a special kind of genius to communicate elegantly with made-up words and be understood

I estimate that it is not unrelated that the same author wrote, "when I use a word, it means precisely what I choose it to mean—neither more nor less."